app/assets/javascripts/components/story/CollapsedStory/CollapsedStory.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import classname from 'classnames';
+import PropTypes from 'prop-types';
+import StoryPopover from '../StoryPopover';
+import StoryDescriptionIcon from '../StoryDescriptionIcon';
+import CollapsedStoryEstimate from './CollapsedStoryEstimate';
+import CollapsedStoryStateActions from './CollapsedStoryStateActions';
+import CollapsedStoryInfo from './CollapsedStoryInfo';
+import CollapsedStoryIcon from './CollapsedStoryIcon';
+
+import {
+  isStoryNotEstimated,
+  isRelease,
+} from '../../../rules/story';
+
+const classNameStory = (storyType, estimate) => classname(
+  'Story',
+  {
+    'Story--unestimated': isStoryNotEstimated(storyType, estimate),
```

Comment:
You could probably refactor this, assigning this call to a const and reusing it.